node/sc/bridge_accounts.go

```diff
@@ -53,13 +53,14 @@ type feePayerDB interface {
 
 // accountInfo has bridge account's information to make and sign a transaction.
 type accountInfo struct {
-	am       *accounts.Manager  // the account manager of the node for the fee payer.
-	keystore *keystore.KeyStore // the keystore of the operator.
-	address  common.Address
-	nonce    uint64
-	chainID  *big.Int
-	gasPrice *big.Int
-	gasLimit uint64
+	am          *accounts.Manager  // the account manager of the node for the fee payer.
+	keystore    *keystore.KeyStore // the keystore of the operator.
+	address     common.Address
+	nonce       uint64
+	chainID     *big.Int
+	gasPrice    *big.Int
+	gasLimit    uint64
+	kip71Config params.KIP71Config
```

Comment:
Is this field used in cAccount also?
If not, it would be nice if the name could be identified as parent kip71 config based on @kjeom 's comment.

Reply:
Never mind this comment.